build.sbt

```diff
@@ -70,6 +76,11 @@ javaOptions ++= Seq("-Xmx512m", "-Djava.awt.headless=true")
 
 scalacOptions ++= Seq("-deprecation", "-unchecked")
 
+testOptions in Test <+= (scalaVersion) map {
```

Comment:
Could you please rewrite this to the new syntax?
E.g. something like `testOptions in Test := scalaVersion.value match ....`